phpBB/adm/style/acp_styles.html

```diff
@@ -32,6 +32,10 @@
 
 <!-- IF L_EXPLAIN --><p>{L_EXPLAIN}</p><!-- ENDIF -->
 
+<fieldset class="quick">
+	<span class="small"><a href="https://www.phpbb.com/customise/db/styles-2" target="_blank">{L_BROWSE_STYLES_DATABASE}</a></span>
```

Comment:
IST sites? Isn't phpBB's styles/ext db the same for all. We won't link to "other" sites other than phpBB's Titania.